.travis.yml

```diff
@@ -18,6 +18,7 @@ env:
   - DB=mysqli
 
 before_install:
+  - sudo sh -c "echo 'precedence ::ffff:0:0/96 100' >> /etc/gai.conf"
```

Comment:
So another `sudo` based workaround just to come around travis issues? =( I was hoping that we could get rid of `sudo` commands totally sometime in the future to make use of docker-based builds and caching...